lib/ui/platform_dispatcher.dart

```diff
@@ -1333,7 +1340,13 @@ class ViewConfiguration {
   /// relative to.
```

Comment:
Not directly related to your PR, but this doc comment doesn't make much sense: "[The] view into which the view is placed"?

Reply:
Similarly, the comment below: What is the "top level view itself"?

Reply:
How about this 75d1cbe7771e687410113c9d64626f2c30b7523a?